js/view/printView.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param {jQuery object} container - references the HTML parent element that contains the view.
+ * @param {Object} model - the reference to the Dinner Model
+ */
+
+var PrintView = function (container, model) {
+    var numberOfGuests = container.find(".numberOfGuests");
+    var orderedItems = container.find("#orderedItemsList");
+
+    var loadOrderedItems = () => {
+        numberOfGuests.text(model.getNumberOfGuests());
+        model.selectedDishes.forEach(e => {
+            var imgAtLeft = $("<img>");
+            var name = $("<h2/>");
+            var description = $("<p/>");
+            var preparation = $("<h3/>");
+            var prepText = $("<p/>");
+            var midPart = $("<section/>");
+            var rightPart = $("<section/>");
+            var listItem = $("<li/>");
+            imgAtLeft.prop("src", "../images/" + e.image);
+            name.text(e.name);
+            description.text(e.description);
+            preparation.text("PREPARATION");
```

Comment:
Writing text in upper-case is a style issue; it is often preferable to not use all-caps in templating.